src/Interactive/EditorFeatures/Core/Implementation/Interactive/InteractiveDocumentSupportsSuggestionService.cs

```diff
@@ -11,6 +14,23 @@ internal sealed class InteractiveDocumentSupportsCodeFixService : IDocumentSuppo
     {
         public bool SupportsCodeFixes(Document document)
         {
+            SourceText sourceText;
+            if (document.TryGetText(out sourceText))
+            {
+                ITextBuffer buffer = sourceText.Container.TryGetTextBuffer();
+                if (buffer != null)
+                {
+                    IInteractiveEvaluator evaluator = (IInteractiveEvaluator)buffer.Properties[typeof(IInteractiveEvaluator)];
+                    IInteractiveWindow window = evaluator?.CurrentWindow;
+                    if (window?.CurrentLanguageBuffer == buffer)
+                    {
+                        // These are only correct if we're on the UI thread.
+                        // Otherwise, they're guesses and they might change immediately even if they're correct.
+                        return !window.IsResetting && !window.IsRunning;
```

Comment:
I forced it to return true and it looks like the fix just silently does nothing.  I'll try again with the debugger attached to try to confirm that we're not leaving things in a bad state.